openedx/core/djangoapps/content/__init__.py

```diff
@@ -2,3 +2,7 @@
 Setup the signals on startup.
 """
 import openedx.core.djangoapps.content.course_structures.signals
+
+# Importing this in this manner seems to resolve some inexplicable circular import issue.
```

Comment:
I'm not sure why, but this issue no longer seems to exist in my testing. I shall remove it.